### DISCUSSION TITLE
Surfacing video comments during playback

### DISCUSSION THREAD

**Discussion by jazzdan** [RecordReplay/devtools#3756] [General]
Watching a video that has comments on the video doesn't feel as useful as it could be. When playing the video I can see the comments on the timeline, which is helpful. But:

1) It's hard to know, while watching, which comment on the timeline corresponds to which comment on the left. I'm guessing it's chronological but my monkey brain has a hard time keeping track.

2) A big component to a video comment is where the comment was placed on that frame of the video, but that isn't represented at all when you're playing the video. It'd be cool if when your playhead reached a comment it like animated on the video frame or something. Maybe with a bullet time effect so you don't miss it? 😛

**Comment by jonbell-lot23** [comment]
March 15, 2022:

Totally agree with these insights. This can be tightened, and it would help.
